src/prism.c

```diff
@@ -5782,10 +5782,21 @@ pm_source_file_node_create(pm_parser_t *parser, const pm_token_t *file_keyword)
     pm_source_file_node_t *node = PM_ALLOC_NODE(parser, pm_source_file_node_t);
     assert(file_keyword->type == PM_TOKEN_KEYWORD___FILE__);
 
+    pm_node_flags_t flags = 0;
+
+    switch (parser->frozen_string_literal) {
+        case PM_OPTIONS_FROZEN_STRING_LITERAL_DISABLED:
+            flags |= PM_STRING_FLAGS_MUTABLE;
```

Comment:
Actually I changed this line to be compatible with string node.

Reply:
Oh shoot, you're right I shouldn't have added that extra case. It shouldn't be marked as static literal, which is what I was fixing.